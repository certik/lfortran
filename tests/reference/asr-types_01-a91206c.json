{
    "basename": "asr-types_01-a91206c",
    "cmd": "lfortran --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/types_01.f90",
    "infile_hash": "551d14053f907f18f3ffcd47309e735206789f8bd754d11187c8fab3",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-types_01-a91206c.stdout",
    "stdout_hash": "a9a5c2ff481c0e7c23a7bb5c06036beed1d7b3be8a65ffeeebe63202",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}