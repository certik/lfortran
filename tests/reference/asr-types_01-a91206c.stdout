(TranslationUnit (SymbolTable 1 {types_01: (Program (SymbolTable 2 {r: (Variable 2 r Local () () Default (Real 4 []) Source Public Required .false.)}) types_01 [] [(= (Var 2 r) (RealConstant   1.00000000000000000e+00 (Real 4 [])) ()) (= (Var 2 r) (RealConstant   1.50000000000000000e+00 (Real 4 [])) ()) (= (Var 2 r) (RealConstant   1.00000000000000000e+00 (Real 4 [])) ())])}) [])
